src/test/java/com/hotel/service/raterule/HotelRateRuleServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package com.hotel.service.raterule;
+
+import com.hotel.adapter.HotelRateRuleAdapter;
+import com.hotel.mappers.common.POSMapper;
+import com.hotel.mappers.rateRule.request.RuleMessageMapper;
+import com.hotel.mappers.rateRule.response.HotelRateRuleResponseMapper;
+import com.hotel.service.common.Context;
+import com.hotel.service.common.ResponseStatus;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.opentravel.ota._2003._05.ArrayOfSourceType;
+import org.opentravel.ota._2003._05.OTAHotelBookingRuleRQ;
+import org.opentravel.ota._2003._05.OTAHotelBookingRuleRS;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.lenient;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.atLeast;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.mock;
+
+@ExtendWith(MockitoExtension.class)
+public class HotelRateRuleServiceTest {
+

```

Comment:
The test class for HotelRateRuleService might added by Team 1. Please check with them and remove if they already added

Reply:
Removed it